docs/proposals/PROPOSAL_J.md

```diff
@@ -0,0 +1,260 @@
+# Proposal J - new media type for compatibilities with update in image index platform
+
+This proposal is born from proposal A to I, providing another option to hold compatibilities, and retriving the compatibilities in shorter path.
+
+The purpose of the proposal, based on the maintainers feedback, is to address the concern maintaining a standalone artifact specification in OCI.
+
+## image spec platform object extension
+
+### revive platform.features
+
+platform.features will be reused for image selection, refer to [proposal E](https://github.com/opencontainers/wg-image-compatibility/blob/main/docs/proposals/PROPOSAL_E.md)
+
+### add platform.compat descriptor
+
+Add a new descriptor in platform object in [image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md#image-index-property-descriptions).
```

Comment:
I have concerns that this descriptor would not be known to existing registries and client tooling. As a result, registries would accept an incomplete manifest, or silently delete the compatibility blob in a later GC cycle. And client tooling to copy images would fail to include the compatibility blobs.

Reply:
yes, this is the risk. some experiments need to be done to investigate the result of unknown descriptor

Reply:
@sudo-bmitch is right, so far I confirmed that with regclient and zot registry. For this proposal we would have to add a new media type support for regclient and zot. Didn't test other registries and tools yet.